xwiki/content.md

```diff
@@ -8,6 +8,31 @@ As an application wiki, XWiki allows for the storing of structured data and the
 
 %%LOGO%%
 
+# Table of contents
+
+<!-- generated with pandoc -f gfm --toc -o readme-toc.md README.md -->
+
+-	[Introduction](#introduction)
+-	[How to use this image](#how-to-use-this-image)
+	-	[Pulling existing image](#pulling-existing-image)
+		-	[Using docker run](#using-docker-run)
+		-	[Using docker-compose](#using-docker-compose)
+		-	[Using Docker Swarm](#using-docker-swarm)
+	-	[Using an external Solr service](#using-an-external-solr-service)
+		-	[Preparing Solr container](#preparing-solr-container)
+		-	[Docker run example](#docker-run-example)
+		-	[Docker Compose example](#docker-compose-example)
+	-	[Building](#building)
+-	[Upgrading XWiki](#upgrading-xwiki)
+-	[Details for the xwiki image](#details-for-the-xwiki-image)
+	-	[Configuration Options](#configuration-options)
+	-	[Passing JVM options](#passing-jvm-options)
+	-	[Miscellaneous](#miscellaneous)
+-	[License](#license)
+-	[Support](#support)
+-	[Contribute](#contribute)
+-	[Credits](#credits)
```

Comment:
This is very unlikely to work nicely on the Hub, so I'd recommend ditching it for now (especially since you're planning to trim this content anyhow).

Reply:
Originally and since I had taken a lot of time to sync the page I wanted to apply so that in case we wanted to go back we had a nice state to go back to.

But it's unlikely so I'll pepare a new PR and I'll close this one. Thanks!